files/templates/scrapeconfigs/additional-scrape-configs.template.yaml

```diff
@@ -1684,6 +1684,131 @@
     action: drop
 [[ include "_common" . | indent 2 ]]
 [[ include "_labelingschema" . | indent 2 ]]
+# Mayu
+- job_name: [[ .ClusterID ]]-prometheus/mayu-[[ .ClusterID ]]/0
+  honor_labels: true
+  scheme: https
+  bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+  tls_config:
+    ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+    insecure_skip_verify: true
+  static_configs:
+  - targets: 
+    - [[ .Mayu ]]:4080
+    labels:
+      app: mayu
+  relabel_configs:
+[[ include "_labelingschema" . | indent 2 ]]
+[[- if eq .Installation "puma" ]]
+- job_name: management-cluster-blackbox-ingress-http
+  honor_labels: true
+  metrics_path: /probe
+  params:
+    module: [upstream_check]
+  static_configs:
+  - targets:
+    - 10.127.3.131:30038
+    - 10.127.3.132:30038
+    - 10.127.3.133:30038
+    - 10.127.3.134:30038
+    - 10.127.3.135:30038
+    - 10.127.3.136:30038
```

Comment:
I'd appreciate if this list could be sorted.

Reply:
Those targets are all failing on puma and the blackbox exporter is no more so I will remove them

Reply:
Sorted